### PR TITLE
Turkish language translation added.

### PR DESCRIPTION
**Pull request by karsibali**
I loved the application. So, I translated it into my native language: Turkish. I hope this helps other people.
Thanks...

### CODE REVIEW

File: safeeyes/safeeyes/config/lang/tr.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+    "meta_info": {
+        "language_name": "Türkçe",
+        "language_name_en": "Turkish"
+    },
+    "exercises": {
```
Comment (slgobinath) [inline safeeyes/safeeyes/config/lang/tr.json]: `exercises` structure is changed to:
`
"exercises": {
        "short_break_close_eyes": "Tightly close your eyes",
        "short_break_roll_eyes": "Roll your eyes",
        "short_break_rotate_clockwise": "Rotate your eyes in clockwise direction",
        "short_break_rotate_counter_clockwise": "Rotate your eyes in counterclockwise direction",
        "short_break_blink": "Blink your eyes",
        "short_break_focus_far_distance": "Focus on a point in the far distance",
        "short_break_drink_water": "Have some water",
        "long_break_walk": "Walk for a while",
        "long_break_lean_back": "Lean back at your seat and relax"
    }
`

File: safeeyes/safeeyes/config/lang/tr.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            "Gözlerinizi saat yönünün tersine döndürün",
+            "Göz kapaklarınızı kapatıp açın",
+            "Uzak mesafede bir noktaya odaklanın",
+            "Gidip biraz su için"
+        ]
+    },
+    "messages": {
+        "ready_for_a_break": "{} Saniye içerisinde bir molaya hazır olun",
+        "safe_eyes_is_disabled": "Safe Eyes etkin değil",
+        "next_break_at_noon": "Bir sonraki mola saat {}'de",
```
Comment (slgobinath) [inline safeeyes/safeeyes/config/lang/tr.json]: All `next_break_at_*` are replaced by a single `"next_break_at": "Next break at {}"`

File: safeeyes/safeeyes/config/lang/tr.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            "Gözlerinizi saat yönünde döndürün",
+            "Gözlerinizi saat yönünün tersine döndürün",
+            "Göz kapaklarınızı kapatıp açın",
+            "Uzak mesafede bir noktaya odaklanın",
+            "Gidip biraz su için"
+        ]
+    },
+    "messages": {
+        "ready_for_a_break": "{} Saniye içerisinde bir molaya hazır olun",
+        "safe_eyes_is_disabled": "Safe Eyes etkin değil",
```
Comment (slgobinath) [inline safeeyes/safeeyes/config/lang/tr.json]: `safe_eyes_is_disabled` is replaced by:
`
"disabled_until_restart": "Disabled until restart",
 "disabled_until_x": "Disabled until {}",
`

File: safeeyes/safeeyes/config/lang/tr.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        "short_break_duration": "Kısa mola süresi (saniye)",
+        "long_break_duration": "Uzun mola süresi (saniye)",
+        "interval_between_two_breaks": "İki mola arasındaki süre",
+        "no_of_short_breaks_between_two_long_breaks": "İki uzun mola arasındaki kısa mola sayısı",
+        "time_to_prepare_for_break": "Mola için hazırlanma zamanı (saniye)",
+        "idle_time": "Minimum boş zaman (dakika)",
+        "strict_break": "Kesin mola (Geç düğmesini gizler)",
+        "audible_alert": "Mola sonunda sesli uyarı",
+        "language": "Dil",
+        "enable": "Safe Eyes etkin",
```
Comment (slgobinath) [inline safeeyes/safeeyes/config/lang/tr.json]: `enable` was earlier a statement like "Safe Eyes is enabled" now it is a command like "Enable Safe Eyes".
Further, there are five more new attributes are added here; which are:
`
"disable": "Disable Safe Eyes",
        "for_x_minutes": "For {} Minutes",
        "for_x_hour": "For {} Hour",
        "for_x_hours": "For {} Hours",
        "until_restart": "Until restart",
`

**Comment by slgobinath** [comment]
Hi @karsibali,
Thank you very much for your contribution. I have modified the format of language file last night. Could you please adapt those changes as well?
I have commented the changes in the review.

Thanks & Regards,
Gobinath

**Comment by slgobinath** [comment]
Thank you @karsibali. The PR is merged.

**Comment by karsibali** [comment]
Thank you....

**Pull request merged**